assets/src/edit-story/components/form/dateTime/timePicker.js

```diff
@@ -193,13 +194,21 @@ function TimePicker({ onChange, is12Hour, localData, setLocalData }) {
     changeDate(newDate, { am: value });
   };
 
+  const hours = useRef();
+  useEffect(() => {
+    hours.current.focus();
+  }, []);
+
+  const isAM = localData.am === 'AM';
+  const isPM = localData.am === 'PM';
   return (
```

Comment:
It might just be me now, but - I could really use a "Today" button around here (it did end up in a completely wrong century though, so my bad):

![Screen Shot 2020-09-28 at 19 16 21](https://user-images.githubusercontent.com/637548/94495229-31c04e80-01bf-11eb-9074-c24584bc742b.png)

Reply:
Yep, probably this could be implemented as part of #1391 although the calendar library itself would also benefit from that, didn't see the option in the props currently but perhaps I'm missing something.